### DISCUSSION TITLE
Is there a way to make PowerPoint templates generated from LibreOffice Impress work with Quarto?

### DISCUSSION THREAD

**Discussion by jorgesca** [quarto-dev/quarto-cli#9822] [Q&A] [answered]
### Description

As a Linux user I usually use LibreOffice whenever required by any reason to provide an MS Office document. In this particular case I am required to use powerpoint, with a specific format, for a presentation. I wrote the presentation in quarto and generated a powerpoint from it. 
The problem appears when I try to create a template presentation with LibreOffice's Impress to suit the required format. I followed the guide in [Guide->Presentations->Powerpoint](https://quarto.org/docs/presentations/powerpoint.html#powerpoint-templates) but fail to get it to work. If I use the template generated from  `quarto pandoc -o template.pptx --print-default-data-file reference.pptx` directly, the presentation is generated correctly, albeit with the moot plain default style. If I make *any* change to this template with Impress, then quarto doesn't generate a proper presentation anymore. 
I know this is probably an issue with how Impress is generating the template (LibreOffice is mostly compatible with MS Office, but quirks are still present, especially when editing PowerPoint files), but maybe there is a way around this problem, and/or maybe quarto can handle LibreOffice's PowerPoint templates with a little bit of effort.
I don't have access to PowerPoint right now, I guess that would be the simplest, but rather inconvenient, way of solving this: editing the template directly in PowerPoint.

**Comment by mcanouil** [comment, marked as answer]
Pandoc (thus Quarto) relies on a very specific PowerPoint template file which uses specific OpenXML.
If your tool does not modify properly the OpenXML for it to still be like the original, then there is nothing that can be done.
PowerPoint templating is limited to what is already inside the template. You cannot add other arbitrary slide structure, etc.

**Comment by jorgesca** [comment]
Thanks for your answer, I guess I was expecting this, but just asked in case there was a workaround.
